features/user_can_see_relevant_info_when_logged_in.feature

```diff
@@ -0,0 +1,19 @@
+Feature: User can see relevant info when logged in
+  As a user,
+  In order to know that I am logged in,
+  I would like to see a welcome message and my profile info
+
+  Background:
+    Given the following user exists
+       | email                | password       | password_confirmation     |
+       | lisa@gmail.com       | heythere123    | heythere123               |
+
+    When I visit the index page
```

Comment:
We could add an 'I should see' 'A social family platform' for the visitor before logging in.